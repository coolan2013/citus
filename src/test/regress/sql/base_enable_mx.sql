--
-- Setup MX data syncing
--
SELECT start_metadata_sync_to_node(:'worker_1_host', :worker_1_port);
SELECT start_metadata_sync_to_node(:'worker_2_host', :worker_2_port);

