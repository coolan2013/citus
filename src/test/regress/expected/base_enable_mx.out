--
-- Setup MX data syncing
--
SELECT start_metadata_sync_to_node('<host>', :worker_1_port);
 start_metadata_sync_to_node
---------------------------------------------------------------------

(1 row)

SELECT start_metadata_sync_to_node('<host>', :worker_2_port);
 start_metadata_sync_to_node
---------------------------------------------------------------------

(1 row)

